CHANGELOG.md

```diff
@@ -2,21 +2,18 @@
 
 ## [0.2.1](https://github.com/lukso-network/tools-eip191-signer/compare/v0.2.0...v0.2.1) (2022-10-28)
 
```

Comment:
There are no release notes for 0.2.2. I guess because this release is just to update the readme on npm?

Reply:
No the reason is that the release for 0.2.2 is done automatically by the bot when we merge the PR into main. 
The bot set automatically the new version on the package.json and the changelog.